code/modules/mob/living/carbon/human/species/slime.dm

```diff
@@ -85,11 +85,16 @@
 
 /datum/species/slime/handle_life(mob/living/carbon/human/H)
 	// Slowly shifting to the color of the reagents
+	if(!H.blood_color) // it starts null, and roundstart slimes sometimes don't change colours, so blood would not line up
```

Comment:
Sounds like this is something that should be fixed in the initialization of the slime instead of in the `handle_life`

Reply:
Adjusted last night, forgot to push, but since things like dye bottles and CMA exists, the check needs to be outside the loop to make sure the blood matches the skin